README

```diff
@@ -151,6 +151,32 @@ To create and play with a Test object from the example package,
 	}
 
 
+Parameter interpretation
```

Comment:
Hmmm. My Sough African / American sense of English must be subtly different. How does  "Interpretation of parameters" look?

Also, the field in plugin.proto is called "parameter" which is even more confusing :-)  https://github.com/google/protobuf/blob/b640f216a56ae25aab9a1fc6f21aa1a56c304494/src/google/protobuf/compiler/plugin.proto#L64